textalyze.js

```diff
@@ -17,7 +17,14 @@ function itemCounts(array) {
   // Your code here.
   // Run 'npm test' to see what tests need to pass.
 
+  array.forEach((value) => {
+    const sum = counts.get(value) ? counts.get(value) + 1 : 1;
+    counts.set(value, sum);
+  });
+
   return counts;
 }
+const array = ['car', 'house', 'car', 'b', 'c', 'd', 'c'];
```

Comment:
Wrap this in

```javascript
if (require.main === module) {
    const array = ...;
}
```

This will allow the code to both be included externally and run directly.

Reply:
Oh, it shouldn't be there, it was just for testing